packages/storage/src/reference.ts

```diff
@@ -201,6 +203,28 @@ export class Reference {
     });
   }
 
+  /**
+   * @param options.maxResults limits the total number of prefixes and items to return..
+   * @param options.pageToken nextPageToken from a previous list(). Resume from that position.
```

Comment:
Maybe change this to:

'The `nextPageToken` from a previous list() response. If provided, listing is resumed from that position.